src/renderer/components/OpenButton/index.jsx

```diff
@@ -42,10 +42,9 @@ export default class LoadButton extends React.Component {
         overlay={<Tooltip>{tipText}</Tooltip>}
       >
         <Button
-          className="mx-3"
+          className={this.props.className}
```

Comment:
Good question, I went back and forth and I actually decided to assign the class on `OpenButton` because that's where the rest of the context for the DOM is. For example I thought "float-right" would be out of context here on `Button`, but would make more sense when also looking at the "d-inline-block" class assigned to the sibling node. But yeah, there's a tradeoff with having to pass it through props.

Reply:
Okay, makes sense